### PR TITLE
Make batchnorm default for UNet

### PR DESCRIPTION
**Pull request by dskhudia**
Instance norm doesn't preserve memory format, i.e., if input is NHWC the output is NWHC. Therefore, instanceNorm pays the cost of NHWC => NCHW while the preceding conv pays the cost of NCHW => NHWC memory format change. 

5.5% improvement in wall_clock_train.

dice (validation): 69.99 (instance norm) 69.33 (batch norm)

### CODE REVIEW

**Review by ravi-mosaicml — APPROVED** [review]
LGTM; from the code perspective. Though maybe wait for an approval from the reserach team?

**Comment by hanlint** [comment]
cc: @bandish-shah and @florescl

**Comment by bandish-shah** [comment]
Changes look good to me but I would prefer @florescl sign off on this since there is a bit of an accuracy regression.

**Comment by florescl** [comment]
@dskhudia would it be possible to do a few more runs to see if the acc regression is just noise?

**Comment by dskhudia** [comment]
@florescl Few more runs with different random seed or the same random seed?

**Comment by dskhudia** [comment]
@florescl 
With instance norm (i.e., without this PR) I see the following dice values for 4 seeds: 68.7, 69.4, 69.61, 70.07

So it seems like the change in dice is within noise margin.

**Review by bandish-shah — APPROVED** [review]
LGTM, thanks for running the multi-seed experiment!

**Review by florescl — APPROVED** [review]
awesome, thanks!

**Pull request merged**
